### PR TITLE
Expand on query-functions and transforms

### PR DESCRIPTION
**Pull request by 4hlberg**

### CODE REVIEW

**Review by gra-moore — CHANGES_REQUESTED** [review]
see comments

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -275,6 +275,59 @@ and to get entities referencing a given entity, e.g. all entities of type person
             --inverse=true
 ```
 
+### Incoming or outgoing query
+
+The query is incoming if you are looking for entities that reference the id of the entity your are starting with in their references and it is outgoing if the starting entity is referencing other entities' ids in its refrences
```
[the remark this answers is not in the document]
Comment (4hlberg) [reply DOCUMENTATION.md]: Rewritten, check it out

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -275,6 +275,59 @@ and to get entities referencing a given entity, e.g. all entities of type person
...
+    "props":{
+        "ns0:a-property":"a"
+    }
+}
+
+>referencing entity
+{
+    "id":"ns2:id-of-referencing-entity",
+    "refs":{
+        "ns1:id":"ns0:starting-entity"
```
[the remark this answers is not in the document]
Comment (4hlberg) [reply DOCUMENTATION.md]: Changed a bit, see if that makes more sense

File: DOCUMENTATION.md [hunk 3]
```diff
@@ -275,6 +275,59 @@ and to get entities referencing a given entity, e.g. all entities of type person
...
+```
+
+#### Outgoing
+
+```json
+>starting entity
+{
+    "id":"ns0:starting-entity",
+    "refs":{
+        "ns1:id":"ns2:id-of-referenced-entity"
```
[the remark this answers is not in the document]
Comment (4hlberg) [reply DOCUMENTATION.md]: See above

File: DOCUMENTATION.md [hunk 4]
```diff
@@ -730,20 +790,33 @@ The Query function is used to lookup related entities. It accepts an array of en
 The result is a list of lists where each inner list is a result row. The result row contains the entity id, the property, and then the related entity. Note: that if an entity has multiple related entities then each appear in its own row.
 
 ```json
-// expand these examples
+>returned from the Query function:
 [
     [ "entity-id" , "property uri", { "id" : "related entity" } ]
```
[the remark this answers is not in the document]
Comment (4hlberg) [reply DOCUMENTATION.md]: Expanded on the structure here, have not changed the wording in the description, but rather clarified with examples.

**Review by gra-moore — APPROVED** [review]

**Pull request merged**